constants/wram_constants.asm

```diff
@@ -324,18 +324,18 @@ DEF PLAYER_SURF      EQU 4
 DEF PLAYER_SURF_PIKA EQU 8
 
 ; wPalFadeMode::
-DEF PALFADE_WHICH        EQU %11
-DEF PALFADE_FLASH_F      EQU 2
-DEF PALFADE_PARTIAL_F    EQU 3
-DEF PALFADE_SKIP_FIRST_F EQU 4
+DEF PALFADE_WHICH       EQU %11
+DEF PALFADE_FLASH_F     EQU 2
+DEF PALFADE_PARTIAL_F   EQU 3
+DEF PALFADE_SKIP_LAST_F EQU 4
```

Comment:
Was this rename because "skip first" was inaccurate, or has the code changed to skip last instead of first?

Reply:
I'll admit, i'll need to look into this further. This came from copying the overworld fading changes from Polished Coral. Speaking of which; I need to make sure we credit pkmncoraldev and Polished Coral for the overworld fading changes.